collector/nodes.go

```diff
@@ -1510,6 +1510,54 @@ func NewNodes(logger log.Logger, client *http.Client, url *url.URL, all bool, no
 				},
 				Labels: defaultNodeLabelValues,
 			},
+			{
+				Type: prometheus.CounterValue,
+				Desc: prometheus.NewDesc(
+					prometheus.BuildFQName(namespace, "ingest", "total_count"),
```

Comment:
Counts should end in `_total`

```suggestion
					prometheus.BuildFQName(namespace, "ingest", "total"),
```